docs/pages/web/text.js

```diff
@@ -185,6 +186,15 @@ export default function TextPage({ generatedDocGen }: {| generatedDocGen: DocGen
           title="Text-wrapping and hyphenation"
```

Comment:
this section is pretty much irrelevant right>

          title="Text-wrapping and hyphenation" Hyphenation on iOS is turned off by default...

Reply:
Hard for me to say, because it's been around for a long time.